core/templates/dev/head/filters.js

```diff
@@ -17,6 +17,8 @@
  */
 
 oppia.constant('RULE_SUMMARY_WRAP_CHARACTER_COUNT', 30);
+oppia.constant('NON_INTERACTIVE_INPUTS', ['collapsible', 'image', 'link',
```

Comment:
This constant shouldn't be added -- I'm pretty sure we're already loading a list of allowed RTE components somewhere. That constant should be used instead. Otherwise people will forget to update this one.